src/nvim/tui/tui.c

```diff
@@ -1719,6 +1728,8 @@ static void augment_terminfo(TUIData *data, const char *term,
     data->unibi_ext.set_cursor_color = (int)unibi_add_ext_str(
         ut, NULL, "\033]12;#%p1%06x\007");
   }
+  data->unibi_ext.reset_cursor_color = (int)unibi_add_ext_str(
+      ut, NULL, "\x1b]112\x07");
```

Comment:
instead of NULL we usually have a descriptive name like "ext. reset_cursor_color".

(Though we forgot to do so for `set_cursor_color`.)